lib/dynamic_config/dcdo.rb

```diff
@@ -55,6 +55,8 @@ def frontend_config
       'student-email-post-enabled': DCDO.get('student-email-post-enabled', false),
       'progress-v2-metadata-enabled': DCDO.get('progress-v2-metadata-enabled', false),
       'show-updated-lms-content': DCDO.get('show-updated-lms-content', false),
+      # Used to test DCDO setting via cookie. See: 'dashboard/test/ui/features/cookie_dcdo.rb'
+      cookie_dcdo_test: DCDO.get('cookie_dcdo_test', nil),
```

Comment:
Why is this necessary to include in the set of values we forward to frontend code?

Reply:
It's just the easiest way to identify that the DCDO cookie stubbing is working,
but we can add a separate tag with data attr to test this.

Reply:
Ahh, of course. Yeah, if possible I'd love to do this in a way that keeps test-specific functionality isolated from things we do for the sake of the user experience in production. Even if that means creating a template we can render just for the sake of the test, like we do for https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/test_dcdo.haml